templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,55 @@ contents:
       echo ""
     }
 
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo "${ip_hint}"
+    }
+
+    # This function waits for ip address set in node-ip service and used as hint
+    # to define br-ex to be bindable
+    # This is workaround for OCPBUGS-673 as it will not allow starting crio
+    # before address is bindable
+    try_to_bind_address() {
+      local ip_hint_file="$1"
+      ip_hint=$(get_ip_from_ip_hint_file "${ip_hint_file}")
+      if [[ -z "${ip_hint}" ]]; then
+        return
+      fi
+      while true; do
```

Comment:
I think this should try for a bounded amount of time instead of forever. An unexpected issue could get us stuck here.